test/integration/stock/import.js

```diff
@@ -75,9 +95,11 @@ describe.skip('test/integration/stock/import The stock import API', () => {
     return agent.post('/stock/import')
       .type('form')
       .field('depot_uuid', params.depot_uuid)
+      .field('date', formattedDate)
       .attach('file', fs.createReadStream(invalidFile))
       .then(res => {
-        helpers.api.errored(res, 400, 'ERRORS.BAD_DATA_FORMAT');
+        const errorMsg = `[line : 2] The inventory group undefined must be a valid text`;
```

Comment:
This is an interesting error message.  Could you make it into something that could be translated into French for our French-speaking users?